drivers/timer/nrf_rtc_timer.c

```diff
@@ -23,23 +23,18 @@
  * to handle
  */
 #define RTC_MIN_DELTA          2
-#define RTC_MASK               0x00FFFFFF
+
 /* Maximum difference for RTC counter values used. Half the maximum value is
  * selected to be able to detect overflow (a negative value has the same
  * representation as a large positive value).
  */
+#define RTC_MASK               0x00FFFFFF
```

Comment:
This should not be moved here, since the comment above is for `RTC_HALF`, not for this one.
Perhaps it would be better to move it to the line after `#define RTC_CC_EVENT`.

Reply:
@anangl: Fixed. Thank you for spotting that. I misunderstood comment and thought that it should be moved.